x/lockup/spec/08_params.md

```diff
@@ -0,0 +1,15 @@
+<!--
+order: 8
+-->
+
+# Parameters
+
+The lockup module contains the following parameters:
+
+| Key                    | Type            | Example |
+| ---------------------- | --------------- | ------- |
+| MinPeriod              | string (uint64) | "86400" |
+| MinAmount              | string (uint64) | "100"   |
```

Comment:
Min amount probably doesn't make sense, because it would be denom specific.

Reply:
removed